newVersion.py

```diff
@@ -544,6 +544,13 @@ def add_common_stage_arguments(parser, new_mobile=True):
     parser.add_argument('-t', '--title', dest='title', help='Release title')
     parser.add_argument('--notes', dest='notes', help='Release notes')
 
+def check_node_version():
```

Comment:
Maybe add a comment that states why we are requiring at least v16

Reply:
See my other comment.

Reply:
Someone looking at this code would think there's something in newVersion.py that requires v16. I just would like to see a comment that says we're requiring a particular version so the package-lock.json remains compatible.

Reply:
Fixed.

Reply:
(I didn't exactly do what you asked for. If you want me to specifically mention package-lock.json, I can do that.)